manifests/pip.pp

```diff
@@ -271,10 +266,18 @@
       }
 
       'latest': {
+        # Unfortunately this is the smartest way of getting the latest available package version with pip as of now
+        $latest_version = join(["${pip_env} install ${proxy_flag} ${pkgname}==notreallyaversion 2>&1",
```

Comment:
We should really ensure that this works with an acceptance test.

Reply:
Sure, I will work on that @bastelfreak .

Reply:
I think this doesn't fetch the packages from a custom index URL if someone happens to use it.